spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -601,30 +646,24 @@ def __init__(self, figure_viewer, parent=None, background_color=None):
 
     def setup_gui(self):
         """Setup the main layout of the widget."""
-        scrollarea = self.setup_scrollarea()
-        up_btn, down_btn = self.setup_arrow_buttons()
-
         layout = QVBoxLayout(self)
         layout.setContentsMargins(0, 0, 0, 0)
         layout.setSpacing(0)
-        layout.addWidget(up_btn)
-        layout.addWidget(scrollarea)
-        layout.addWidget(down_btn)
+        layout.addWidget(self.setup_scrollarea())
 
     def setup_scrollarea(self):
         """Setup the scrollarea that will contain the FigureThumbnails."""
         self.view = QWidget()
 
         self.scene = QGridLayout(self.view)
         self.scene.setContentsMargins(0, 0, 0, 0)
-        self.scene.setSpacing(3)
+        self.scene.setSpacing(5)
```

Comment:
@jnsebgosselin pleas add a comment here to have a clearer idea of what this does.

Reply:
I removed `self.scene.setSpacing(5)` since 5 is already the default value, so there is no need to set it explicitly.